console/console-init/ui/src/utils/utils.ts

```diff
@@ -131,9 +131,10 @@ const kFormatter = (num: number) => {
 };
 
 const uniqueId = () => {
```

Comment:
Don't see the purpose of creating this method. It's simply returning lodash unique id. Why not use lodash unique id directly?

Reply:
In future we might have to remove lodash from the dependency. since lodash is not using elsewhere apart from uniqueId

Reply:
@ajaypratap003 correct me if i am wrong

Reply:
@anukritijha As I said that we should use lodash but indirectly , I mean in utils uniqueId function. As we know that we are using very limited `lodash`  functions. In case we will have to remove` lodash`  lib due to any reason in that case we will have to change in each file but if we will use lodash in utils function for uniqueId then we will have to change at one place. 
Anyways lodash uniueId will use through utils functon. Nothing is logical change here. Hope you will get my point.
@anukritijha wdyt?

Reply:
@ajaypratap003 @drcoolsanjeev Makes sense.